script.js

```diff
@@ -0,0 +1,11 @@
+const toggleMenu = document.querySelector(".navbar__menu__icon");
+const toggleMenuIcon = document.querySelector(".navbar__menu__icon--content");
```

Comment:
`toggleMenuIcon` and `toggleList` variables can be in function body. Now they are unnecessarily global.